wear/src/main/AndroidManifest.xml

```diff
@@ -102,6 +105,18 @@
             <meta-data android:name="androidx.wear.tiles.PREVIEW"
                 android:resource="@drawable/template_tile_example" />
         </service>
+        <service
+            android:name=".tiles.SearchTile"
```

Comment:
Should we name it `ConversationTile` ?

Reply:
yea let me replace `Search` with `Conversation` throughout to make it easier :)